### PR TITLE
Http2 dos mitigations

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

File: pkg/package-server/server/server.go [hunk 1]
```diff
@@ -194,6 +196,11 @@ func (o *PackageServerOptions) Run(ctx context.Context) error {
 		log.SetLevel(log.DebugLevel)
 	}
 
+	// Enables http2 DOS mitigations for unauthenticated clients.
+	utilfeature.DefaultMutableFeatureGate.SetFromMap(map[string]bool{
+		string(genericfeatures.UnauthenticatedHTTP2DOSMitigation): true,
+	})
+
```
Comment (awgreene) [inline pkg/package-server/server/server.go:203]: Does everyone think this is the right space to make the function call?

Comment (tmshort) [reply pkg/package-server/server/server.go:203]: Looks as reasonable as anyplace.

**Review by tmshort — APPROVED** [review]
/lgtm

**Comment by zdtsw** [comment]
Do you have plan to release a newer version to include this fix?

**Comment by awgreene** [comment]
A release was cut yesterday, https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/v0.26.0

**Pull request merged**